tools/shims/Tests/Unit/src/ShimsTests.ts

```diff
@@ -115,8 +115,16 @@ export class ShimsTests extends AITestClass {
                 QUnit.assert.ok(strShimObject === "object", "check object");
                 QUnit.assert.ok(strShimUndefined === "undefined", "check undefined");
                 QUnit.assert.ok(strShimPrototype === "prototype", "check prototype");
-                QUnit.assert.ok(strShimHasOwnProperty === "hasOwnProperty", "check hasOwnProperty");
             }
         });
+
+        this.testCase({
```

Comment:
Just adding tests so that unexpected changes from ts-utils don't break this functionality